Generator/Sources/SafeNameGenerator.swift

```diff
@@ -0,0 +1,289 @@
+// Sourced from https://github.com/apple/swift-openapi-generator/blob/84b693f9d0559dc488e691edb4837bafbce2aaea/Sources/_OpenAPIGeneratorCore/Translator/TypeAssignment/SafeNameGenerator
```

Comment:
We should probably add this to a `NOTICE.txt` file, I'm happy to do it as a follow up as part of #4 though. Here's Swift OTel's notice file as an example: https://github.com/swift-otel/swift-otel/blob/main/NOTICE.txt